Topic_3/src/keyPoint1/AmountToTextTest.java

```diff
@@ -0,0 +1,111 @@
+package keyPoint1;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class AmountToTextTest {
+
+	@Test
+	public void testForZero() {
+		CheckAmount checkAmount = new CheckAmount(0);
+		String textCheck = checkAmount.toText();
+		assertEquals("Zero and 0/100 dollars", textCheck);
+	}
+
+	@Test
+	public void testForOnes() {
+		CheckAmount checkAmount = new CheckAmount(1);
+		String textCheck = checkAmount.toText();
+		assertEquals("One and 0/100 dollars", textCheck);
+		checkAmount.set(5);
+		textCheck = checkAmount.toText();
+		assertEquals("Five and 0/100 dollars", textCheck);

```

Comment:
“have only one logical assertion per test”. 
Remember, unit tests are a design specification of how a certain behavior should work, not a list of observations of everything the code happens to do.